forem/forem
# Resourcify Badge Achievements

## What type of PR is this? (check all applicable)

## Description
`resourcify` badge achievements so we can allow `user.add_role :single_resource_admin, BadgeAchievement` to be a thing. Needed for internal operations mostly.

Also cleans up the constants' syntax a bit for easier writing

## Related Tickets & Documents
https://github.com/orgs/forem/projects/27#card-55779138

## QA Instructions, Screenshots, Recordings
1. Visit `/admin/users/1/edit`
2. Add the Badge and BadgeAchievement roles
3. See that they've been added properly

### UI accessibility concerns?
Nope

## Added tests?


## [Forem core team only] How will this change be communicated?


## [optional] Are there any post deployment tasks we need to perform?
No

## [optional] What gif best describes this PR or how it makes you feel?

![The letter B](https://media.giphy.com/media/26gscSULUcfKU7dHq/giphy.gif)
